sympy/matrices/matrices.py

```diff
@@ -1268,26 +1268,66 @@ def eigenvals(self, error_when_incomplete=True, **flags):
             return [simplify(value) for value in eigs]
 
     def eigenvects(self, error_when_incomplete=True, iszerofunc=_iszero, **flags):
-        """Return list of triples (eigenval, multiplicity, basis).
-
-        The flag ``simplify`` has two effects:
-            1) if bool(simplify) is True, as_content_primitive()
-            will be used to tidy up normalization artifacts;
-            2) if nullspace needs simplification to compute the
-            basis, the simplify flag will be passed on to the
-            nullspace routine which will interpret it there.
+        """Return list of triples (eigenval, multiplicity, eigenspace).
 
         Parameters
         ==========
 
-        error_when_incomplete : bool
+        error_when_incomplete : bool, optional
             Raise an error when not all eigenvalues are computed. This is
             caused by ``roots`` not returning a full list of eigenvalues.
 
-        If the matrix contains any Floats, they will be changed to Rationals
-        for computation purposes, but the answers will be returned after being
-        evaluated with evalf. If it is desired to removed small imaginary
-        portions during the evalf step, pass a value for the ``chop`` flag.
+        iszerofunc : function, optional
+            Specifies a zero testing function to be used in ``rref``.
+
+            Default value is ``_iszero``, which uses naive and fastest approach
+            by using sympy's default assumption handler.
+
+            It can also accept any user-specified zero testing function, if it
+            is formatted as a function which accepts a single symbolic argument
+            and returns ``True`` if it is tested as zero and ``False`` if it
+            is tested as non-zero, and ``None`` if it is undecidable.
+
+        simplify : bool or function, optional
+            If ``True``, ``as_content_primitive()`` will be used to tidy up
+            normalization artifacts.
+
+            It will also be passed to and interpreted by the ``nullspace``
+            routine.
+
+        chop : bool or positive number, optional
+            If the matrix contains any Floats, they will be changed to Rationals
+            for computation purposes, but the answers will be returned after
+            being evaluated with evalf. If it is desired to removed small
+            imaginary portions during the evalf step, pass a value for the
+            ``chop`` flag.
+
+        Returns
+        =======
+        ret : [(eigenval, multiplicity, eigenspace), ...]
+            A ragged list containing tuples of data obtained by ``eigenvals``
+            and ``nullspace``.
+
+            ``eigenspace`` is a list containing ``eigenvector`` s, obtained by
+            computing ``nullspace`` in regards of each eigenvalues. Be careful
```

Comment:
I think the first line could alone suffice ending with "...with regard to each eigenvalue".